client/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "client",
+  "version": "1.0.0",
+  "scripts": {
+    "dev": "parcel watch index.html",
+    "test": "jest",
+    "build": "npm install --include=dev && parcel build index.html"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "devDependencies": {
+    "jest": "^27.5.1",
+    "parcel": "^2.3.1",
+    "path-browserify": "^1.0.1"
+  },
+  "dependencies": {
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-router-dom": "^6.2.1"
+  }
```

Comment:
Dere må legge til jest collectCoverageFrom korrekt for å få full uttelling for tester på eksamen.